### ISSUE TITLE
Mapped command incorrect for CtrlPMixed

### ISSUE THREAD

**Issue by ristillu** [ctrlpvim/ctrlp.vim#458]
I have set up CtrlP with:
```
let g:ctrlp_cmd = 'CtrlPMixed'
let g:ctrlp_map = '<C-p>'
```
This opens up CtrlP in what says is mixed mode. The prompt I get is:
```
prt  path  <mix>={ files }=<buf> <->
```
but when I instead invoke CtrlP with
```
:CtrlPMixed
```
I get a different result set and a different prompt. The prompt looks like:
```
prt  path  <mru>={ fil + mru + buf }=<fil> <->
```

This behavior is the behavior I want, but I don't want to have to invoke it every time by typing the full command. Is there a way to enable this so it works through my shortcut?

**Comment by alphaCTzo7G** [comment]
@ristillu, 
This is most likely not a problem with `ctrlp`, but maybe with your binding. 

I have `<leader>f` bound to `:CtrlPMixed`. So when I do:

`:map <leader>f

it shows

```
,f            :CtrlPMixed<CR>
```

Can you show us the output of `:map <c-p>`?

**Comment by ristillu** [comment]
Hey, you're right. The binding was right but I didn't RTFM and had tried to set it in .gvimrc. Turns out something happens to initialize CtrlP by the time .gvimrc comes up so even though I set g:ctrlp_map and g:ctrlp_cmd, in .gvimrc, ctrlp.vim had already been read and therefore those variables had no effect.

Solution was to move my CtrlP customization to .vimrc

Thanks for pointing me in the right direction.
